scripts/docs.py

```diff
@@ -135,11 +136,13 @@ def format_op(op):
 
 
 def gen_docs(entrypoint: str, modules: List[str], maintenance: str = "Core"):
-    per_module = {name: [] for name in modules}
+    per_module = {name: [None, []] for name in modules}
```

Comment:
I was going to tell you to try to avoid importing the package directly, but I didn't know how to explain so I just did it to make sure it worked and then it was done so I pushed.